gatsby-node.esm.js

```diff
@@ -1,17 +1,13 @@
-import path from "path";
+import format from "date-fns/format";
 import { createFilePath } from "gatsby-source-filesystem";
-import { Octokit } from "@octokit/core";
+import fetch from "node-fetch";
 import { graphql } from "@octokit/graphql";
```

Comment:
@ comes before A-Za-z

Reply:
Oh, ok. I have just been ignoring @ symbols and going off of the first letter.